src/encoded/tests/features/customsteps.py

```diff
@@ -55,6 +55,26 @@ def click_element(browser, css):
     elements.first.click()
 
 
+@when(parse('I click all elements with the css selector "{css}"'))
+def click_element(browser, css):
```

Comment:
Let me know if this is a concern. I based this code off of "I click element with the css selector {css}"; but I excused the assert statement.